File: src/main/java/org/idea_sp/psi/SourcePawnTokenType.java
package org.idea_sp.psi;

import com.intellij.psi.tree.IElementType;
import org.idea_sp.SourcePawnLanguage;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class SourcePawnTokenType extends IElementType {
    public SourcePawnTokenType(@NotNull @NonNls String debugName) {
        super(debugName, SourcePawnLanguage.INSTANCE);
    }

    @Override
    public String toString() {
        return "SourcePawnTokenType." + super.toString();
    }
}

File: src/main/kotlin/org/idea_sp/psi/SourcePawnTokenType.kt
package org.idea_sp.psi

import com.intellij.psi.tree.IElementType
import org.idea_sp.SourcePawnLanguage
import org.jetbrains.annotations.NonNls

class SourcePawnTokenType(@NonNls debugName: String) : IElementType(debugName, SourcePawnLanguage) {
    override fun toString(): String = "SP(${super.toString()})"
}
